src/node/desktop/src/main/gwt-callback.ts

```diff
@@ -902,10 +903,31 @@ export class GwtCallback extends EventEmitter {
     });
 
     ipcMain.handle('desktop_startup_error_info', async (event, varName: string) => {
+      if (varName == 'launch_failed') {
```

Comment:
Use `===` in Javascript for comparisons. `==` does type coercion and can give unexpected results.